pkg/limayaml/validate.go

```diff
@@ -406,9 +406,6 @@ func warnExperimental(y *LimaYAML) {
 	if *y.MountType == VIRTIOFS && runtime.GOOS == "linux" {
 		logrus.Warn("`mountType: virtiofs` on Linux is experimental")
 	}
-	if *y.VMType == VZ {
-		logrus.Warn("`vmType: vz` is experimental")
-	}
```

Comment:
This is inconsistent with other places where you only add "will graduate in Lima 1.0". I don't care though, as it will need to be removed before the final 1.0 release anyways.

Reply:
This inconsistency arises because users read the docs on the master branch, but they do not execute the code on the master branch.